lib/script.py

```diff
@@ -39,14 +39,15 @@ def generate_products(self):
         if Configuration.current.verbose:
             verbose_flags += "-v"
         verbose_flags += "\n"
-        swift_triple = Configuration.current.target.swift_triple
+        triple = Configuration.current.target.triple
         base_flags = """
-TARGET                = """ + Configuration.current.target.triple + """
+TARGET                = """ + triple + """
 DSTROOT               = """ + Configuration.current.install_directory.absolute() + """
 """
+        swift_triple = triple if triple == "armv7-none-linux-androideabi" else Configuration.current.target.swift_triple
```

Comment:
It seems like this should actually not be logic here but instead live in `lib/target.py` instead